// react-navbar/build/Item.js
'use strict';

Object.defineProperty(exports, "__esModule", {
    value: true
});

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

var _reactIntl = require('react-intl');

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

exports.default = _react2.default.createClass({
    displayName: 'Item',

    propTypes: {
        title: _react2.default.PropTypes.string.isRequired,
        href: _react2.default.PropTypes.string.isRequired,
        target: _react2.default.PropTypes.string
    },

    mixins: [_reactIntl.IntlMixin],

    render: function render() {
        var _props = this.props,
            href = _props.href,
            target = _props.target,
            title = _props.title;


        var anchorProps = Object.assign({ href: href }, target ? { target: target } : {});

        return _react2.default.createElement(
            'li',
            null,
            _react2.default.createElement(
                'a',
                anchorProps,
                this.getIntlMessage(title)
            )
        );
    }
});
